### PR TITLE
more abductee objectives

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

Adds more objectives to abductees, for more wacky antics and uncharacteristic hijinks

### Why is this change good for the game?

I like variety and I think these could be fun.

:cl:  
rscadd: more abductee objectives 
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
<Everything is so freezing cold! Who turned up the air conditioning? Get everything as hot as possible.
just so you know, this abductee objective will be used to plasmaflood the station and kill everyone

**Comment by UselessTheremin** [comment]
> <Everything is so freezing cold! Who turned up the air conditioning? Get everything as hot as possible.
> just so you know, this abductee objective will be used to plasmaflood the station and kill everyone

theres already one to get rid of all the oxygen in the station

**Comment by UselessTheremin** [comment]
but yeah we dont need more of that i will change it

File: strings/abductee_objectives.txt [hunk 1]
```diff
@@ -37,4 +37,20 @@ Your body must be improved. Ingest as many drugs as you can.
...
+The AI is secretly your long lost brother.
+You are a creature of the great below. To channel your inner ocean, drench yourself in water.
+The skeleton inside of you wants to be free. Help it do so.
+Everything is so freezing cold! Who turned up the air conditioning? Get everything as hot as possible.
+Modern medicine is a fallacy, and you hate liars! Make a new kind of medicine to show those big-shot surgeons who's boss around here.
+The plants are watching you. They are evil spies, sent to destroy the station.
+You cannot bring yourself to say any vowels, no matter how hard you try.
+Today is opposite day, everything you know to be true is reversed.
+All of the animals are about to hatch into massive dragons. Help them in doing so.
+Become a monkey man, humanity is too much for you.
```
Comment (alexkar598) [inline strings/abductee_objectives.txt:52]: This kinda sucks, genetics will usually be glad to help you and after that you're kinda just... stuck as a monkey

File: strings/abductee_objectives.txt [hunk 2]
```diff
@@ -37,4 +37,20 @@ Your body must be improved. Ingest as many drugs as you can.
...
+You are a living nuclear bomb, acquire the disk and find a place to detonate yourself to destroy the station.
+Your blood is made of ice cream. You are the ice cream man, and you must give people ice cream.
+Beat up all those nerds on this station! No poindexters allowed! After the nerds are gone, have a cool party with all the jocks.
```
Comment (alexkar598) [inline strings/abductee_objectives.txt:55]: Not a fan of the grief everyone in science this implies

File: strings/abductee_objectives.txt [hunk 3]
```diff
@@ -37,4 +37,20 @@ Your body must be improved. Ingest as many drugs as you can.
...
+Every single person is full of candy. Get the candy from their insides.
```
Comment (alexkar598) [inline strings/abductee_objectives.txt:56]: Nor am I a fan of kill and gib everyone you find and eat their organs

**Comment by UselessTheremin** [comment]
yeah i dont really know what i was thinking with this pr lmao

**Pull request closed without merging**